modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/language.js

```diff
@@ -68,6 +68,45 @@ export function sub(langKey, args, join = true) {
 	return join ? keyArray.join('') : keyArray;
 }
 
+/**
+ * Returns localized information used to link to a resource, like Liferay Learn
+ * articles. The json object `learnMessages` contains the messages and urls.
+ *
+ * Example:
+ * getLocalizedLearnMessageObject("general", {
+ *	"general": {
+ *		"en_US": {
+ *			"message": "Tell me more",
+ *			"url": "https://learn.liferay.com/"
+ *		}
+ *	}
+ * })
+ * => {
+ *			"message": "Tell me more",
+ *			"url": "https://learn.liferay.com/"
+ *		}
+ *
+ * @param {string} resourceKey Identifies which resource to render
+ * @param {Object} learnMessages Contains the messages and urls
+ * @param {string} locale
+ * @param {string} defaultLocale
```

Comment:
Since these are optional parameters, it'd be good to document that or else it might seem like it's a requirement which seems really annoying to have to define every time.

We can use brackets to denote optional parameters like:
```
@param {string} [locale]
``` 

And show default values like: 
```
@param {string} [locale=Liferay.ThemeDisplay.getLanguageId()]
```

You can read more about it here: https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
Oh I didn't know that! Thanks, just added both changes in 👍